packages/clerk-js/src/ui/hooks/usePopover.ts

```diff
@@ -18,13 +18,22 @@ type UsePopoverProps = {
 
 export const usePopover = (props: UsePopoverProps = {}) => {
   const [isOpen, setIsOpen] = React.useState(props.defaultOpen || false);
-  const { reference, floating, strategy, x, y, context } = useFloating({
+  const { update, reference, floating, strategy, x, y, context } = useFloating({
     open: isOpen,
     onOpenChange: setIsOpen,
     whileElementsMounted: props.autoUpdate === false ? undefined : autoUpdate,
     placement: props.placement || 'bottom-start',
     middleware: [offset(props.offset || 6), flip(), shift()],
   });
+
+  useEffect(() => {
+    setIsOpen(props.defaultOpen || false);
```

Comment:
If we end up using this useEffect, lets replace setIsOpen with one of the `toggle`, `open` or `close` functions defined at lines 39-41